polaris-migrator/src/migrations/scss-remove-unused-import/scss-remove-unused-import.ts

```diff
@@ -0,0 +1,58 @@
+import type {FileInfo, API, Options} from 'jscodeshift';
+import postcss from 'postcss';
+import type {Plugin} from 'postcss';
+
+import {createInlineComment} from '../../utilities/sass';
+import {POLARIS_MIGRATOR_COMMENT} from '../../constants';
+
+interface PluginOptions extends Options {
+  importPath?: string;
+  namespace?: string;
+}
+
+const plugin = (options: PluginOptions = {}): Plugin => {
+  const {
+    importPath = 'global-styles/legacy-polaris-v8',
+    namespace = 'legacy-polaris-v8',
```

Comment:
This option shouldn't be necessary, correct? The namespace can be extracted from the `atUseURL`:
https://github.com/Shopify/polaris/pull/7322/files#diff-2d9c77c4c87bf51828a048bdd0a559a0657560a0e7656eaa80932079afe3f68dR94-R95